docs/docs/firmware/state-io-esp32.md

```diff
@@ -2,229 +2,63 @@
-
-Cupim drumstick swine venison short loin capicola bacon picanha spare ribs tongue burgdoggen pork chop cow ham. Shank brisket ball tip tri-tip cow jerky shoulder chicken, corned beef alcatra filet mignon. Biltong jerky picanha ham hock swine. Ham hock shank corned beef, porchetta strip steak venison sirloin turducken hamburger ham rump jerky. Frankfurter pork tri-tip beef ribs meatloaf picanha pastrami doner burgdoggen.
-
----
-
-### How does it work?
-Shoulder picanha tongue cow kevin. Pig alcatra chicken jowl, turducken flank buffalo bresaola burgdoggen ground round venison shoulder pancetta. Sirloin chislic beef ribs turkey biltong. Ground round fatback frankfurter meatball, venison jowl capicola ham hock meatloaf. 
-
-Buffalo boudin meatball ball tip spare ribs. Venison cow pig shank pastrami, chislic ham. Beef ribs hamburger sirloin brisket, capicola drumstick cupim sausage shank burgdoggen swine.
-
-| Col 1         | Col 2        | Col 3      |
-| :------------ |:-------------| :--------- |
-| Row data     | Row data      |  Row data  |
-| Row data     | Row data      |  Row data  |
-| Row data     | Row data      |  Row data  |
-
-Sirloin chislic beef ribs turkey biltong. Ground round fatback frankfurter meatball, venison jowl capicola ham hock meatloaf. 
+In the example above the FW is configured for 32 inputs and 96 outputs. The user has to make sure that the connected MCPs are configured properly (correct MCP address). Not all output MCPs need to be populated. The FW is checking the for installed MCPs automatically and allows configuration (indexes) for existing MCPs only
 
 ## Configuration
-### Sub Title
-Pig alcatra chicken jowl, turducken flank buffalo bresaola burgdoggen ground round venison shoulder pancetta.
-
-```sh
-command-example.sh
-```
-
-Some attribute text:
-- `Attribute here`: Explanation Text here
-- `Attribute here`: Explanation Text here
-- `Attribute here`: Explanation Text here
-
-
-#### Code Examples
-
-Cupim drumstick swine venison short loin capicola bacon picanha spare ribs tongue burgdoggen pork chop cow ham.
-
-```c
-void setup() {
-  // put your setup code here, to run once:
-  Serial.begin(9600);
-  Serial.println("Hello World!");
-}
+This FW is fully compatible with [OXRS](https://oxrs.io) eco-system and is built based on it's   [Libraries](/docs/libraries/README.md)
 
-void loop() {
-  // put your main code here, to run repeatedly:
+All the functionality of the [StateMonitor](/docs/firmware/state-monitor-esp32.md) applies to the input partition and [StateController's](/docs/firmware/state-controller-esp32.md) functionality applies to the output partition. Please see there for further details.

```

Comment:
`and the [StateController] functionality`
`Please see their documentation for further details.`